src/tpm2_pytss/ESAPI.py

```diff
@@ -2331,6 +2390,71 @@ def hmac_start(
 
         return ESYS_TR(sequence_handle[0])
 
+    def mac_start(
+        self,
+        handle: ESYS_TR,
+        auth: Union[TPM2B_AUTH, bytes, str],
+        hash_alg: TPM2_ALG,
+        session1: ESYS_TR = ESYS_TR.PASSWORD,
+        session2: ESYS_TR = ESYS_TR.NONE,
+        session3: ESYS_TR = ESYS_TR.NONE,
+    ) -> ESYS_TR:
+        """Invoke the TPM2_MAC_Start command.
+
+        This function invokes the TPM2_MAC_Start command in a one-call
+        variant. This means the function will block until the TPM response is
+        available.
+
+        Args:
+            handle (ESYS_TR): Handle of an MAC key.
+            auth (Union[TPM2B_AUTH, bytes, str]): Authorization value for subsequent use of the sequence.
+            hash_alg (TPM2_ALG): The hash algorithm to use for the MAC.
+            session1 (ESYS_TR): A session for securing the TPM command (optional). Defaults to ESYS_TR.PASSWORD.
+            session2 (ESYS_TR): A session for securing the TPM command (optional). Defaults to ESYS_TR.NONE.
+            session3 (ESYS_TR): A session for securing the TPM command (optional). Defaults to ESYS_TR.NONE.
+
+        Raises:
+            TypeError: If a parameter is not of an expected type.
+            ValueError: If a parameter is not of an expected value.
+            TSS2_Exception: Any of the various TSS2_RC's the lower layers can return.
+
+        Returns:
+            An ESYS_TR handle of ESYS resource for TPMI_DH_OBJECT.
+
+        C Function: Esys_MAC_Start
+
+        TPM Command: TPM2_MAC_Start
```

Comment:
Same for this command, make sure the attribute exists.

Reply:
@malikabhi05 here do the libversion check and raise NotImplementedError if it's too old.